content/blog/2023/2023_guide_to_speaking.md

```diff
@@ -0,0 +1,55 @@
+Title: Guide to Speaking at PyCon India
+Category: 2023
+Slug: guide-to-speaking
```

Comment:
Please add all the required metadata, you can see in the README what all is required.